cmd/agent/collector.go

```
@@ -158,6 +160,8 @@ func (l *Collector) run(exit chan bool) {
 	// Channel to announce new features detected
 	featuresCh := make(chan features.Features, 1)
 
+	// We poll the features a first time. We will try several times until we receive a response.
+	// So these features cannot change over time, Why don't we provide them into the helm chat instead of using this polling?
```

we need to remove this before merging

---

Actually, we can simplify this logic failing fast if we cannot reach the /feature endpoint